dspace-api/src/main/java/org/dspace/app/util/DCInputsReader.java

```diff
@@ -639,4 +700,8 @@ public String getInputFormNameByCollectionAndField(Collection collection, String
         }
         throw new DCInputsReaderException("No field configuration found!");
     }
+
+    public Map<String, List<String>> getMappedValuePairs() {
```

Comment:
Please add Javadocs to describe the structure of the "MappedValuePairs"

Reply:
good catch! at the end it was an unused caching structure introduced in an early stage of the implementation... I have removed it